langchain/utilities/wikipedia.py

```diff
@@ -65,12 +65,12 @@ def _page_to_document(self, page_title: str, wiki_page: Any) -> Document:
         main_meta = {
             "title": page_title,
             "summary": wiki_page.summary,
+            "source": wiki_page.url,
```

Comment:
that's why i kinda like having both. "source" is generic enough that we could have convention (and eventually enforcement) that every loaded doc has it, "page_url" is specific and makes it easy to get what you're looking for when you're specifically using the wiki loader

Reply:
There is actually a kind of enforcement, because if you want to do `index.query_with_sources()`, de document **must** have `metadata:source`.
And talking about generalization, when I load documents using SeleniumURLLoader document loader, the source value is actually an URL.
I'm here to make whichever change you decide.

Reply:
Pre-change:
![Captura desde 2023-05-10 09-01-18](https://github.com/hwchase17/langchain/assets/25377374/b5630b13-aa00-4fec-a4de-f9a48d5beb2f)
Post-change:
![Captura desde 2023-05-10 09-06-39](https://github.com/hwchase17/langchain/assets/25377374/392465ed-a3a3-411f-bdb1-e3f4f073f959)